config-connector/solutions/apps/helm/wordpress/charts/wordpress-gcp/templates/k8s-external-load-balancer.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  namespace: default
```

Comment:
Do we want to put it to the "default" namespace or in the current namespace?
If creating it in "default" is needed, then in the instructions in the readme, we should ask users to run `get svc wordpress-external --namespace default`

Reply:
Current namespace, good catch. Otherwise we need to add namespace to settings and customize all configs with this namespace, which is not what we are doing for this solution. I've removed reference to default.